src/components/shared/UserAvatar.tsx

```diff
@@ -0,0 +1,26 @@
+import { Avatar } from '@mui/material';
+
+interface Props {
+    avatarUrl?: string | null;
+    userName: string;
+    size?: number;
+}
+
+const DEFAULT_SIZE = 25;
+
+function UserAvatar({ avatarUrl, size, userName }: Props) {
+    return (
+        <Avatar
+            src={avatarUrl ?? ''}
+            sx={{
+                fontSize: (size ?? DEFAULT_SIZE) / 1.5,
```

Comment:
It may be worthwhile to save the result of the common evaluation in a constant and use that value to set `fontSize`, `height`, and `width`.

Reply:
I will... I wasn't sure if it was worth it wile doing but if you noticed it as well I will change it.